src/views/v2/day/time-separator.php

```diff
@@ -24,10 +24,10 @@
 }
 
 $event_start_datetime = strtotime( tribe_get_start_date( $event->ID, true, Tribe__Date_Utils::DBDATETIMEFORMAT ) );
```

Comment:
We should use `$event->dates->start->format( Dates::DBDATETIMEFORMAT )`.
Also: `use Tribe__Date_Utils as Dates` at the top of the file and make the code cleaner.